### ISSUE TITLE
How do I get metastring ?

### ISSUE THREAD

**Issue by NearCloser** [pngwn/MDsveX#146]
is this basic mdsvex config ?
 when writing code in MDX.
I want to get title props...

javascript  title={ MDX is cool }

**Comment by pngwn** [comment]
Can you show me a bit more please, I'm not sure what I'm looking at here.

**Comment by nako0215** [comment]
ok !

I add mdsvex and now I can write mdx.
now I want to use code block with metastring in mdx like this.
    ```javascript  {3,1-5}
(function someDemo() {
  var test = "Hello World!";
  console.log(test);  
})();  

return () => <App />;   
    ```

I heard that code like "{3,1-5}" is called metastring.
https://prince.dev/highlight-with-react

do I able to do get this props called metastring like this site shows ?

**Comment by pngwn** [comment]
Ah I see, I'll take a look at this.

**Comment by pngwn** [comment]
Released in mdsvex@0.9.8!

**Comment by NearCloser** [comment]
> Released in mdsvex@0.9.8!

Thanks!   i wil try it !!
